frontend/src/app/new-todo/new-todo.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+
+@Component({
+  	selector: 'app-new-todo',
+  	templateUrl: './new-todo.component.html',
+  	styleUrls: [ './new-todo.component.css' ]
+})
+export class NewTodoComponent implements OnInit
+{
+
+	todo_object : {title : string, _id : string} = { title:'', _id:'' };
```

Comment:
Just todos... and we never use underscore in JavaScript... it is a camleCase language...